controllers/managedfusionoffering_controller.go

```diff
@@ -192,8 +193,41 @@ func (r *ManagedFusionOfferingReconciler) own(resource metav1.Object) error {
 
 // All the below functions are placeholder for offering plugin integration
 
+type dfOfferingConfig struct {
+	usableCapacityInTiB     int
+	onboardingValidationKey string
+}
+
 // This function is a placeholder for offering plugin integration
 func pluginReconcile(r *ManagedFusionOfferingReconciler) (ctrl.Result, error) {
+	var validationErrorExist bool
+	var dfOfferingConfigData dfOfferingConfig
+
+	r.Log.Info(fmt.Sprintf("Validating %s offering CR config", r.managedFusionOffering.Spec.Kind))
+	if usableCapacityInTiBAsString, found := r.managedFusionOffering.Spec.Config["usableCapacityInTiB"]; !found {
+		r.Log.Error(fmt.Errorf("%s offering CR config does not have a usableCapacityInTiB entry", r.managedFusionOffering.Spec.Kind),
+			"missing usableCapacityInTiB info in offering CR config")
+		validationErrorExist = true
```

Comment:
I think you are mixing the purpose of the error with the one for the message message
```suggestion
		r.Log.Error(
			fmt.Errorf("missing field: usableCapacityInTiB"), 
			"an error occurred while validating Data Foundation ManagedFusionOffering config",
		),
		valid = false
```

Reply:
Updated